firefox-ios/firefox-ios-tests/Tests/ClientTests/Telemetry/HomepageTelemetryTests.swift

```diff
@@ -37,4 +37,27 @@ final class HomepageTelemetryTests: XCTestCase {
         let resultValue = try XCTUnwrap(GleanMetrics.Homepage.privateModeToggle.testGetValue())
         XCTAssertEqual(resultValue[0].extra?["is_private_mode"], "false")
     }
+
+    func testGleanWrapperIntegration() throws {
+        // Define a mock GleanWrapper
+        class MockGleanWrapper: GleanWrapperProtocol {
```

Comment:
We already have a `MockGleanWrapper` class in our tests target. We should use the one that already exists 🙏

Reply:
resolved